Engine/Graphics/CMakeLists.txt

```diff
@@ -73,9 +73,9 @@ list(APPEND GRAPHICS_HEADERS
     Direct3D/RenderD3D.h
     Direct3D/TextureD3D.h
 )
-else()
-include_directories(${SDL_INCLUDE_DIRS})
 endif()
 
+include_directories(${SDL_INCLUDE_DIR})
```

Comment:
could you move that to sdl2.cmake  so this is available project-wide?

Reply:
done